src/lib.rs

```diff
@@ -420,6 +420,14 @@ pub mod type_object;
 pub mod types;
 mod version;
 
+#[deprecated(since = "0.19.0")]
+/// A write-once cell mediated by the Python GIL.
+// FIXME: We want to deprecate this, but that does not yet work for re-exports,
+// c.f. https://github.com/rust-lang/rust/issues/30827
+pub mod once_cell {
+    pub use crate::sync::{GILOnceCell, Interned};
+}
+
```

Comment:
Maybe this should be `doc(hidden)`? It might confusing having both show up in docs.

Reply:
Yes, I agree. I also added the missing deprecation note if this suddenly becomes effective via a rustc change.